Dockerfile

```diff
@@ -18,8 +19,9 @@ RUN apt-get update \
     zlib1g-dev
 
 RUN apt -y update
```

Comment:
nit: Can this be removed safely? It seems pointless to have this here, since there are no new repos being added, and the apt-get update command is already run in line 7.